src/wallet/wallet.cpp

```diff
@@ -1690,18 +1699,19 @@ void CWalletTx::GetAmounts(std::list<COutputEntry>& listReceived,
 
         // In either case, we need to get the destination address
         CTxDestination address;
+        const CScript& scriptPubKey = StripClaimScriptPrefix(txout.scriptPubKey);
```

Comment:
I don't think we want this. We don't want to count our claims as spendable change. The old wallet didn't have it.

Reply:
Not sure what you mean.  https://github.com/lbryio/lbrycrd/blob/legacy_master/src/wallet/wallet.cpp#L1152

Reply:
My bad. I was looking in the wrong spot.